### PR TITLE
📜 Implement the Proposal history section

### LINKED ISSUE #913: Base Proposal History
Implement the history sidebar
(Part of #801)

### PR DESCRIPTION
**Pull request by thesan**
Closes #913
See  #801

### CODE REVIEW

**Comment by thesan** [comment]
@jodator currently [the tooltip which shows the block number](https://www.figma.com/file/pgNl83wgAub186krVkdiuo/Proposals?node-id=267%3A3163) gets out of the screen, but I'm thinking of solving this in another PR

**Comment by thesan** [comment]
> @jodator currently [the tooltip which shows the block number](https://www.figma.com/file/pgNl83wgAub186krVkdiuo/Proposals?node-id=267%3A3163) gets out of the screen, but I'm thinking of solving this in another PR

Looks like this has already been solve

File: packages/ui/test/app/pages/ProposalPreview.test.tsx [hunk 1]
```diff
@@ -45,12 +45,17 @@ describe('ProposalPreview', () => {
...
+
+    const proposerSection = (await findByText(sideBar, 'Proposer')).parentElement as HTMLElement
+    expect(proposerSection).toBeDefined()
+    expect(await findByText(proposerSection, MEMBER_ALICE_DATA.handle)).toBeDefined()
+
+    expect(await findByText(sideBar, 'History')).toBeDefined()
 
     for (const name of ['Approved', 'Rejected', 'Slashed', 'Abstained', 'Not Voted']) {
-      expect(await screen.findByRole('heading', { name })).toBeDefined()
+      expect(await findByRole(sideBar, 'heading', { name })).toBeDefined()
```
Comment (jodator) [inline packages/ui/test/app/pages/ProposalPreview.test.tsx:58]: Could you use the `within( sidebar ).findBy..` form? It's more fluent than `findByRole( element, ... )` form.

**Review by jodator — CHANGES_REQUESTED** [review]
Looks good, just a few comments to check 👍

File: packages/ui/test/app/pages/ProposalPreview.test.tsx (hunk 1 above)
Comment (thesan) [reply packages/ui/test/app/pages/ProposalPreview.test.tsx]: Nice :astonished: I didn't know about this form

**Pull request merged**